### ISSUE TITLE
Error when trying to start a task without closing the previous task window

### ISSUE THREAD

**Issue by auwtch** [jedrzejboczar/toggletasks.nvim#3]
First of all thanks for the great plugin! Integrates beautifully with existing tools.
The error occurs if I start the second task directly from another task window (without closing it).

Error:
```
E5108: Error executing lua ...packer/opt/telescope.nvim/lua/telescope/actions/init.lua:348: Invalid window id: 1038
stack traceback:
        [C]: in function 'nvim_win_get_cursor'
        ...packer/opt/telescope.nvim/lua/telescope/actions/init.lua:348: in function 'run_replace_or_original'
        ...k/packer/opt/telescope.nvim/lua/telescope/actions/mt.lua:65: in function 'key_func'
        ...ack/packer/opt/telescope.nvim/lua/telescope/mappings.lua:242: in function 'execute_keymap'
        [string ":lua"]:1: in main chunk
```

Steps to reproduce:
1. `require("telescope").extensions.toggletasks.spawn()` and start some task
2. Do not close the current window, just open the spawn window in step 3
3. `require("telescope").extensions.toggletasks.spawn()` and start another task
4. *an error occurs*

**Comment by jedrzejboczar** [comment]
Thanks for the report. I did some debugging and it looks like it only occurs when using floating toggleterm windows. This is what happens:

1. We are in "task1" floating window opened by the first spawn.
2. Calling `spawn()` starts telescope picker.
3. Telescope first [saves original_win_id](https://github.com/nvim-telescope/telescope.nvim/blob/1a91238a6ad7a1f77fabdbee7225692a04b20d48/lua/telescope/pickers.lua#L342) which is the id of "task1" window.
4. Then Telescope opens its windows, which triggers `WinLeave` from "task1" window, which [closes "task1" floating window](https://github.com/akinsho/toggleterm.nvim/blob/93c2f2ca96af74f22f470e2fbcbd5e89b1faa63d/lua/toggleterm.lua#L113-L115).
5. After selecting new task to spawn [we call action.close(prompt_buf)](https://github.com/jedrzejboczar/toggletasks.nvim/blob/4329ad580799f25c0a923a2d1e71a585ae0bbc48/lua/toggletasks/telescope.lua#L174) to first close telescope windows (that's how telescope actions have to be implemented).
6. Telescope then [tries to restore the original window](https://github.com/nvim-telescope/telescope.nvim/blob/1a91238a6ad7a1f77fabdbee7225692a04b20d48/lua/telescope/actions/init.lua#L347-L348), but this is "task1" window which has already been closed in 4.

So it looks like this is a telescope issue. I'll create a PR with a fix in telescope repo.
